app/js/canopy-exporter.js

```diff
@@ -29,7 +29,16 @@
     targetArray[offset + 3] = (aNumber >> 24) & 255;  // byte 4
   }
 
-  // TODO: Does 32 bit float work??
+  // Return the bits of the float as a 32-bit integer value.  This
+  // produces the raw bits; no intepretation of the value is done.
+  function _floatBits (f) {
```

Comment:
Is this the only way of doing the conversion? Seems very wasteful. We are creating 3 arrays for 1 value.